proto/run/v2/run.proto

```diff
@@ -49,6 +53,9 @@ enum SpanStepOp {
   INVOKE = 1;
   SLEEP = 2;
   WAIT_FOR_EVENT = 3;
+  STEP = 4;
```

Comment:
**question**: What is the difference between `STEP` and `RUN`?

Reply:
`RUN` is the function run itself, `STEP` is the steps within the run.

Reply:
`RUN` is the function run itself, `STEP` is the steps within the run.

Reply:
Thanks - that was misleading to me.